js/packages/seed-vault/android/settings.gradle

```diff
@@ -0,0 +1,7 @@
+dependencyResolutionManagement {
```

Comment:
This can be removed now. I originally was using the version catalog in the SV JS package but that broke stuff so it was reverted. remove this.